### PR TITLE
Mock data added to client for all APT GET requests

### PR DESCRIPTION
**Pull request by NylonEric**
* added mock data for products API GET requests
* added mock data for Reviews API GET requests
* added mock data for questions and answers GET API requests
* added mock data for cart GET API request
* data imported into ProductDetailPage.jsx in the data type it returns from API GET requests (array/object)
* tests for mock data types

### CODE REVIEW

**Comment by milofultz** [comment]
dont morge yet, I gotta delete all my dummy data

File: client/__tests__/MockData.test.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+test('Should load the mock data for the questions and answers', () => {
+  expect(typeof questionsMock).toBe('object');
+  expect(typeof answersMock).toBe('object');
+});
+
+test('Should load the mock data for the reviews', () => {
+  expect(typeof reviewsMock).toBe('object');
+  expect(typeof reviewsMetaMock).toBe('object');
+});
```
Comment (milofultz) [inline client/__tests__/MockData.test.js:40]: good tests!

File: client/src/components/ProductDetailPage.jsx [hunk 2]
```diff
@@ -44,6 +49,9 @@ class ProductDetailPage extends React.Component {
         <RelatedItemsAndOutfit />
         <QuestionsAndAnswers />
         <RatingsAndReviews onChangeFileHandler={this.onChangeFileHandler} onClickUploadHandler={this.onClickUploadHandler} />
+        <div>{console.log(`Test of mock data passed to render function:\n Mock Products: ${productsMock} \n
```
Comment (milofultz) [inline client/src/components/ProductDetailPage.jsx:52]: Should probably remove this console log now, if we know the mock data works

**Review by milofultz — COMMENTED** [review]
Just that one console.log nit

File: client/src/components/ProductDetailPage.jsx (hunk 2 above)
Comment (NylonEric) [reply client/src/components/ProductDetailPage.jsx:52]: Will do. I'll resubmit after removing.

**Review by milofultz — APPROVED** [review]

**Pull request merged**